ImportStats v1.js

```diff
@@ -239,7 +244,7 @@
         }
 
         // Power
-        regex = /(?:#|\.\s+)([A-Z][\w-]+(?:\s(?:[A-Z][\w-]+|[\(\)\d/-]|of)+)*)(?=\s*\.)/g;
+        regex = /(?:#\s*)([A-Z]['\w-]+(?:\s(?:[A-Za-z]['\w-]+|\([\w\d/]+\)|-|of)+)*)(?=\s*\.)/g;

```

Comment:
This fixes a few problems:
1. It allows (1/day) type traits
2. It allows traits that have lower case words like "and" or "with"
3. It allows traits with `'` in them